README.rst

```diff
@@ -54,6 +54,8 @@ dockcross/linux-arm64
   |linux-arm64-images| Cross compiler for the 64-bit ARM platform on Linux,
   also known as AArch64.
 
+dockcross/linux-arm-cortex
```

Comment:
Based off the [information here](https://collaborate.linaro.org/display/TCWGPUB/ARM+and+AArch64+Target+Triples), this toolchain should be called `dockcross/bare-armv7` to be consistent with the other toolchain names.